BAUDRATE: 2000000           #串口波特率
control_cycle: 0.002        #电机控制周期

joints_number: 7            #单台机械臂的关节数量（目前支持10个即两组机械臂）
arms_number: 2              #机械臂数量（包括主和从）

Gripper_with_current: true

MasterDxl:                  #主机械臂1
  DXL_ID: [1,2,3,4,5,6,7]             #电机ID
  DEVICE_NAME: /dev/ttyUSB0       #串口号（支持每台机械臂使用相同或不同串口号）

SlaveDxl:                   #从机械臂1
  DXL_ID: [1,2,3,4,5,6,7]
  DEVICE_NAME: /dev/ttyUSB1